kvmagent/kvmagent/plugins/network_plugin.py

```
@@ -600,6 +771,276 @@ def delete_bonding(self, req):
+
+    def _init_lldpd(self, config_file):
+        r, lspci_output = bash_ro("lspci | grep -i -E 'eth.*X710|eth.*X722|eth.*XL710'")
+        lldp_stop_command = "lldp stop\n"
+
+        if r == 0:
+            for line in lspci_output.splitlines():
+                nic_pci_address = line.split()[0]
+                nic_pci_address_path = '/sys/kernel/debug/i40e/0000:%s' % nic_pci_address
+                if os.path.exists(nic_pci_address_path):
+                    command_file_path = os.path.join(nic_pci_address_path, "command")
+                    if not os.path.exists(command_file_path) or lldp_stop_command not in open(command_file_path).read():
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)